### PR TITLE
add global scheduler internal API demo docs

### PR DESCRIPTION
**Pull request by nkwangjun**
> /kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

issues #81 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by nkwangjun** [comment]
@cathyhongzhang  This is used for understanding api design and the new scheduler processes, thanks

File: docs/design-proposals/global-scheduler/GlobalSchedulerInternalAPIDemo.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+     ],
+     "selected": {
+      "node_id": "site1",
+      "region": "region1",
+      "availability_zone": "az1"
+     }
+    }
+   ]
+  }
+ }
```
Comment (cathyhongzhang) [inline docs/design-proposals/global-scheduler/GlobalSchedulerInternalAPIDemo.md:105]: LGTM

**Review by cathyhongzhang — APPROVED** [review]

File: docs/design-proposals/global-scheduler/GlobalSchedulerInternalAPIDemo.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+Means there is one volume pool, and it is in region1 and az1, and available volume capacity is 2000 gb.
+
+You should change scheduler_policy_file path as your environment in conf.yaml
```
[the remark this answers is not in the document]
Comment (cathyhongzhang) [reply docs/design-proposals/global-scheduler/GlobalSchedulerInternalAPIDemo.md:50]: @jshaofuturewei I see that your comment has been addressed. Could you change it to resolved?

**Comment by nkwangjun** [comment]
Please merge to master, thanks. @cathyhongzhang

**Review by jshaofuturewei — APPROVED** [review]

**Pull request merged**